provision/Provision-StreamAnalyticsJob.ps1

```diff
@@ -120,10 +136,63 @@ $storageAccountKey = Get-AzureStorageKey -StorageAccountName $StorageAccountName
 $storageAccountKeyPrimary = $storageAccountKey.Primary
 $RefdataContainerName = $ContainerName+"refdata"
 
-$JobDefinitionText = (Get-Content -LiteralPath $JobDefinitionPath).
-                    Replace("_StreamAnalyticsJobName",$StreamAnalyticsJobName).
+# Create SQL Job Definition
+
+
+$JobDefinitionText = (Get-Content -LiteralPath $JobDefinitionPathSQL).
+                    Replace("_StreamAnalyticsJobName",$StreamAnalyticsSQLJobName).
+                    Replace("_Location",$Location).
+                    Replace("_ConsumerGroupName",$ConsumerGroupNameSQL).
+                    Replace("_EventHubName",$EventHubName).
+                    Replace("_ServiceBusNamespace",$ServiceBusNamespace).
+                    Replace("_EventHubSharedAccessPolicyName",$EventHubSharedAccessPolicyName).
+                    Replace("_EventHubSharedAccessPolicyKey",$EventHubSharedAccessPolicyKey).
+                    Replace("_AccountName",$StorageAccountName).
+                    Replace("_AccountKey",$storageAccountKeyPrimary).
+                    Replace("_Container",$ContainerName).
+                    Replace("_RefdataContainer",$RefdataContainerName).
+                    Replace("_DBName",$SqlDatabaseName).
+                    Replace("_DBPassword",$SqlDatabasePassword).
+                    Replace("_DBServer",$SqlServerName).
+                    Replace("_DBUser",$SqlDatabaseUser)
+
+$TempFileName = [guid]::NewGuid().ToString() + ".json"
+
+$JobDefinitionText > $TempFileName
+
+$VerbosePreference = "SilentlyContinue" 
+Switch-AzureMode AzureResourceManager
+$VerbosePreference = "Continue" 
+
+New-AzureResourceGroupIfNotExists -ResourceGroupName $ResourceGroupName -Location $Location
```

Comment:
This call is duplicate. Both jobs are in the same resource group. We should keep only the first call.